extmod/network_nina.c

```diff
@@ -0,0 +1,591 @@
+/*
```

Comment:
I think this file should be `network_ninaw10.c` to match the config variable `MICROPY_PY_NETWORK_NINAW10`.